### ISSUE TITLE
Tilemap becomes white when bundled

### ISSUE THREAD

**Issue by subsoap** [defold/editor2-issues#1574]
What it looks like when ran from the editor

![2018-01-13 10_06_18-starode](https://user-images.githubusercontent.com/409170/34908740-735e59ea-f849-11e7-8c12-a6db6854431d.png)

When bundled

![2018-01-13 10_06_28-starode](https://user-images.githubusercontent.com/409170/34908743-8141e216-f849-11e7-935f-a8b5358b88ed.png)

![ground](https://user-images.githubusercontent.com/409170/34908747-92030fda-f849-11e7-9e80-d0954422525d.png)

![2018-01-13 10_07_51-defold editor 2 0 - starode](https://user-images.githubusercontent.com/409170/34908752-a8b62294-f849-11e7-941f-5b7a4e311277.png)

![2018-01-13 10_08_17-defold editor 2 0 - starode](https://user-images.githubusercontent.com/409170/34908759-b8ad90d8-f849-11e7-9e39-a2a4b1fca337.png)

**Comment by subsoap** [comment]
I changed texture to be a power of 2 and !!!!
![2018-01-13 10_15_00-starode](https://user-images.githubusercontent.com/409170/34908795-a8ca2824-f84a-11e7-96ad-ffdc72628a44.png)

![ground](https://user-images.githubusercontent.com/409170/34908799-ae57ede4-f84a-11e7-84da-2b876a9e10aa.png)

**Comment by andsveking** [comment]
Do you get the same issue when you compare run/bundle from Ed1?

**Comment by matgis-king** [comment]
Also, does it exhibit the same issue when building if you Enable Texture Compression from the Editor 2 preferences? It's possible that some texture compression schemes only work with power-of-two texture dimensions.

**Comment by subsoap** [comment]
Even with "Enable Texture Compression" checked in Editor 2 with non-power of 2 tilemap it's still white in bundle, and weird glitch when the png uses indexed colors instead of RGB. It looks right bundled when power of 2 and RGB.

I'll download editor 1 again soon to test non-power of 2 tilemap texture.

**Comment by subsoap** [comment]
Update: Also tested non-power of 2 + RGB instead of indexed colors and it's not white, doesn't look wrong. So looks like indexed color pngs are the problem not necessarily non-power of 2.

**Comment by matgis-king** [comment]
Thanks for checking! To clarify, it works fine if you choose **Project > Build** in Editor 2 even if "Enable Texture Compression" is checked?

**Comment by subsoap** [comment]
Yes, it works with and without "Enable Texture Compression" in Editor 2 so long as the png is not indexed colors and instead is RGB.
